benchmarks/nel/scripts/datasets/dataset.py

```diff
@@ -393,11 +400,58 @@ def evaluate(
             eval_results.append(spacyfishing_results)
 
         logger.info(dict(cand_gen_label_counts))
-        evaluation.EvaluationResults.report(tuple(eval_results))
+        evaluation.EvaluationResults.report(tuple(eval_results), run_name=run_name, dataset_name=self.name)
 
         self._nlp_best.config["incl_context"] = False
         self._nlp_best.config["incl_prior"] = False
 
+    def compare_evaluations(self, highlight_criterion: str) -> None:
+        """Generate and display table for comparison of all available runs for this dataset.
```

Comment:
Can you add a comment here about how candidate generation rows are treated separately? It is slightly confusing, but the data is similar enough I think it should be OK if there's an explanation somewhere.

Reply:
Extended the description. Please resolve this conversation if you think it's ok (and feel free to approve the PR if you think nothing else is to be changed).